### PR TITLE
qemu: add smartcard support

### PR DESCRIPTION
**Pull request by Sir-Boops**
First build will fail.

### CODE REVIEW

**Comment by Sir-Boops** [comment]
The i686 build timed, so everything seems good

File: srcpkgs/qemu/template [hunk 1]
```diff
@@ -19,7 +19,7 @@ makedepends="libpng-devel libjpeg-turbo-devel pixman-devel snappy-devel
  libbluetooth-devel libssh2-devel libusb-devel pulseaudio-devel libnfs-devel
  $(vopt_if sdl SDL-devel) $(vopt_if sdl2 SDL2-devel) $(vopt_if gtk gtk+-devel)
  $(vopt_if spice spice-devel) $(vopt_if virgl virglrenderer-devel)
- $(vopt_if opengl 'libepoxy-devel libdrm-devel MesaLib-devel')"
+ $(vopt_if opengl 'libepoxy-devel libdrm-devel MesaLib-devel') libcacard-devel"
```
Comment (Gottox) [inline srcpkgs/qemu/template]: Can you add it as a build_option?

Comment (Sir-Boops) [inline srcpkgs/qemu/template]: Added as a default for i686 & x86_64

Comment (Duncaen) [inline srcpkgs/qemu/template]: Do we need this by default, I really dont want it.

Comment (Sir-Boops) [inline srcpkgs/qemu/template]: No, we really don't need it as a default option, i only added it as one as i wasn't aware some don't want it. I'll remove it from the default build options.

**Comment by Sir-Boops** [comment]
Put adding libcacard into it's own commit

**Comment by pullmoll** [comment]
For consistency please use `New package: libcacard-2.5.2` as the first commit message.

File: srcpkgs/libcacard/template [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'libcacard'
+pkgname=libcacard
+version=2.5.2
+revision=1
+build_style=gnu-configure
+hostmakedepends="pkg-config"
+makedepends="nss-devel glib-devel pcsclite-devel"
+short_desc="A library to privide emulation of smart cards to a virtual card reader"
```
Comment (ebfe) [inline srcpkgs/libcacard/template]: typo

**Pull request merged**